src/helpers/git_helpers/rebase_helpers.sh

```diff
@@ -30,3 +30,11 @@ function rebase_in_progress {
     echo false
   fi
 }
+
+
+# Determines what branch is being rebased
+#
+# Only call when rebase_in_progress is true
+function rebasing_branch_name {
+  sed 's/^refs\/heads\///' < .git/rebase-apply/head-name
```

Comment:
yep. I'm uncertain what to do otherwise though. I will merge this with get_current_branch_name so that it checks if there is a rebase in progress and if so returns the rebasing_branch_name otherwise returns the branch name as we normally get it.